packages/styles/components/Input.module.scss

```diff
@@ -0,0 +1,40 @@
+@import '~@venice/styles/core';
+
+.container {
+  display: flex;
+  flex-flow: column;
+  .label {
+    font: var(--typography-caption-font);
+    margin-bottom: var(--space-xs);
+    color: #616161;
+  }
+  .input {
+    background: var(--disabled-color-contrast);
+    border: 1px solid #ebebeb;
+    box-sizing: border-box;
+    border-radius: var(--border-radius);
+    font: var(--typography-body-font);
+    padding: var(--space-sm);
+
+    &:disabled,
+    &[disabled] {
+      background: #e7e7e7;
```

Comment:
we have a brand to disabled items: https://juntossomosmais.github.io/venice/react/?path=/docs/getting-started-styles-colors--page

Reply:
This color is very different from the color of Figma, is there a possibility to change the color of the brand?